src/data/thunks.js

```diff
@@ -152,7 +152,7 @@ export function startTimedExam() {
       return;
     }
     await updateAttemptAfter(
-      exam.course_id, exam.content_id, createExamAttempt(exam.id),
+      exam.course_id, exam.content_id, createExamAttempt(exam.id, exam.use_legacy_attempt_api),
```

Comment:
we could just pass the whole exam in here instead of creating an new argument. I chose not to do this because it's an untyped dictionary and I dislike passing around a dict of mystery contents instead of explicit args. Also, I'd hope this gets removed with an edx-proctoring DEPR anyway.